docker/dev/find_fastest_go_proxy.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Define a list of mirrors without using Bash arrays.
+# c.f. https://stackoverflow.com/a/23930212/724872
+DEFAULT_URL="https://pkg.go.dev/"
+read -r -d '' GO_PROXIES <<EOM
+${DEFAULT_URL}
+https://goproxy.cn/,direct
```

Comment:
How about we don't write `https://` and `,direct` here?  If we can do so, we can reuse a Bash function `find_fastest_url` and we don't need to call `cut` to retrieve the URL to the left.